Purchases/RCAttributionFetcher.m

```diff
@@ -56,7 +56,7 @@ - (nullable NSString *)identifierForVendor
 
 - (void)adClientAttributionDetailsWithCompletionBlock:(void (^)(NSDictionary<NSString *, NSObject *> * _Nullable attributionDetails, NSError * _Nullable error))completionHandler
 {
-#if TARGET_OS_IPHONE
+#if TARGET_OS_IOS
```

Comment:
`TARGET_OS_TV` won't enter here and it was before, is that expected

Reply:
`ADClient` isn't available on apple tv https://developer.apple.com/documentation/iad/adclient?language=objc
(actually, the entire iAd isn't available on TV for whatever reason)

Reply:
thinking about this more, maybe the ideal way to do this is to set up a couple of macros that define: 
`PLATFORM_INCLUDES_IAD` and stuff like that so that the intention is very clear. thoughts?